scripts/bed_coverage.py

```diff
@@ -16,7 +16,7 @@
 
 bed_filenames = sys.argv[1:]
 if bed_filenames:
-    input = chain(* (open(_) for _ in bed_filenames))
+    input = chain.from_iterable(map(open, bed_filenames))
```

Comment:
What about:

```suggestion
    input = fileinput.input(bed_filenames)
```

and replace:
```python
import sys
from itertools import chain
```

with:
```python
import fileinput
import sys
```

Reply:
This can be much better, but I wanted to be careful in my refactoring. If you'd like to take over this PR and add these changes, open a new one and tag this one, and I'll close it.

Reply:
Sure, thanks for opening the PR in the first place! If you can confirm my proposed changes work in your case, I'll do the necessary work and merge it.

Reply:
To be honest I never ran `bx-python`. I've been going over lots of repos fixing the `chain` bug, and I figured the tests will make sure it's okay. On this PR, I see there's an unrelated flake8 error...